xdg_shell.h

```diff
@@ -34,6 +33,7 @@ struct cg_xdg_popup {
 	struct wl_listener reposition;
 };
 
+struct cg_xdg_shell_view *xdg_shell_view_from_view(struct cg_view *view);
```

Comment:
Once the comment above is resolved, this no longer needs to be declared in this header